borderless/explainer.md

```diff
@@ -0,0 +1,512 @@
+
+Mock of app settings UI with app’s origin:  
+![App settings UI mock](./images/app-settings-ui.png)
+
+### Privacy indicators
+
+Removing the title bar also removes the area where critical privacy indicators
+would live, e.g. for camera and microphone access. However there is already
+another on-going project to move those privacy indicators to the shelf.
+
+![Example privacy indicator](./images/privacy-indicators-in-shelf.png)
+
+See (Googlers only)
+[go/cros-privacy-indicators-design](go/cros-privacy-indicators-design) for more.
+
+## Other notable details
+
+### “Settings and more” three-dot button
+
+Since the three-dot button (which gives users access to extensions, security
+information about the page, access to cookies, etc.) will be disappearing, the
+most critical information (app’s origin) will be displayed in the app settings.
+
+The menu behind the three-dots button contains the following:
+
+<img src="./images/three-dot-menu.png" alt="Three-dot menu" width="450">
+
+<table>
+  <tr>
+   <td><strong>Button</strong></td>
+   <td><strong>Status</strong></td>
+  </tr>
+  <tr>
+   <td>App info</td>
+   <td>
+    <ul>
+      <li>App’s origin will be added to App settings.</li>
+      <li>Cookies will be only accessible through developer console.</li>
+      <li>Connection security will be only accessible through developer console.</li>
+    </ul>
+   </td>
+  </tr>
+  <tr>
+   <td>Extensions</td>
+   <td>Will not be supported in the first version. This is something that can be considered and implemented later if needed.
+   </td>
+  </tr>
+  <tr>
+   <td>Copy URL</td>
+   <td>Will not be supported in the first version. This is something that can be considered and implemented later if needed.
+   </td>
+  </tr>
+  <tr>
+   <td>Open in Chrome</td>
+   <td>Will not be supported in the first version. This is something that can be considered and implemented later if needed.
+   </td>
+  </tr>
+  <tr>
+   <td>Zoom</td>
+   <td>Will still work using shortcuts (Ctrl++ & Ctrl+- & Ctrl+0).</td>
+  </tr>
+  <tr>
+   <td>Print</td>
+   <td>Will still work using shortcuts (Ctrl+P).</td>
+  </tr>
+  <tr>
+   <td>Find</td>
+   <td>Will still work using shortcuts (Ctrl+F).</td>
+  </tr>
+  <tr>
+   <td>Cast</td>
+   <td>Will not be supported in the first version. This is something that can be considered and implemented later if needed.
+   </td>
+  </tr>
+  <tr>
+   <td>Cut</td>
+   <td>Will still work using shortcuts (Ctrl+X).</td>
+  </tr>
+  <tr>
+   <td>Copy</td>
+   <td>Will still work using shortcuts (Ctrl+C).</td>
+  </tr>
+  <tr>
+   <td>Paste</td>
+   <td>Will still work using shortcuts (Ctrl+V).</td>
+  </tr>
+</table>
+
+A way to get the three-dots menu working would be that it could be part of the
+AWC feature, but that discussion should rather be part of the AWC feature. This
+could also be something considered to implement later on if seen necessary.
+
+## Demo
+
+Flags to enable:
```

Comment:
Instructions for how to get the demo working with Chrome flags should probably be in a ChromeOS-specific document, and not here.

I think it's appropriate to keep links to your demo sites though, to illustrate the API.

Reply:
I will move these to the run instructions of the demo-app. :) Any case these should be removed when everything is actually launched.